src/SesWebhookCall.php

```diff
@@ -20,4 +20,15 @@ public static function storeWebhook(WebhookConfig $config, Request $request): We
             'payload' => $message->toArray(),
         ]);
     }
+
+    public static function getFirstIdOfThisSesMessage(string $ses_message_id): int
```

Comment:
It seems we're only using this function in the `isFirstOfThisSesMessage()` call below, I'd opt to just inline the query in that function

Reply:
I was having a problem with the test. When this query was in isFirstOfThisSesMessage(), the query brought back a string, not an integer during the test run.

Letting the return type of getFirstIdOfThisSesMessage() do the conversion to int seemed better than seemingly unnecessarily calling intval() for the conversion.

I think it's a general issue with using sqlite for the testing testing suite. Any insight on that?

Reply:
`(int) SesWebhookCall::where('payload->MessageId', $ses_message_id)->min('id')` should do exactly the same as the type conversion on the method

Reply:
Ill get that change committed Ina bit once I get off the bus to work.

It's unfortunate that what should be an int already needs to be cast to int just for the sake of testing.

Are there examples of this in other spatie packages?

Reply:
No example I can think of off the top of my head but I know Eloquent models sometimes come back with string IDs as well, the explicit type cast is absolutely no issue, and makes the intent pretty clear on what we're expecting